frame/composable-traits/src/nft/mod.rs

```diff
@@ -0,0 +1,112 @@
+//! Default interface used to interact with financial NFTs through a NFT provider.
+//!
+//! Allows  allows to enter positions and sell it before position is left as NFT.
+//! Allows to split financial positions.
+//!
+//! # Design
+//! There are 2 ways to integrate NFT and financial positions.
+//!
+//! Store all metadata of position in NFT BLOB or store only reference in NFT.
+//!
+//! What we have in case of storing reference:
+//! - NFT account produced from identity is set as owner of position
+//! - protocol providing position implements trait which allow to work with it through NFT API
+//! - position is stored in any way efficient for that
+//! - NFT stores only reference to position in protocol, example pallet identifier and some
+//!   monotonous identifier
```

Comment:
what `monotonous identifier` means?
I met `strict monotonic a counter/sequence` only, f.e. [Learn You Some Erlang for great good!](https://learnyousomeerlang.com/time#on-time-for-time)